### DISCUSSION TITLE
Api order creation with the context for a custom relationship

### DISCUSSION THREAD

**Discussion by LucasLaurens** [Sylius/Sylius#14917] [General]
Hello everyone, 

I am working with the api and I have created an endpoint that needs to create a command but it is not directly linked to it. To do this, in my handler I use the getCart of the cartContext. I wanted to know if this was a good practice or if it was better to use the factory and pass the right properties to the endpoint?

**Comment by LucasLaurensMLP** [comment]
History:

<img width="390" alt="Capture d’écran 2023-04-07 à 09 25 39" src="https://user-images.githubusercontent.com/91264443/230562701-7c9119b1-294c-4168-b648-bc8e24ff560a.png">
